File: Backend/Models/Utilizatori.cs
using System;
using System.Collections.Generic;

#nullable disable

namespace Relmonitor.Models
{
    public partial class Utilizatori
    {
        public Utilizatori()
        {
            Releases = new HashSet<Release>();
        }

        public int Idutilizator { get; set; }
        public string Nume { get; set; }
        public string Email { get; set; }
        public int Esteadmin { get; set; }
        public DateTime Datacreare { get; set; }
        public DateTime Datamodificare { get; set; }

        public virtual ICollection<Release> Releases { get; set; }
    }
}

File: Frontend/src/app/models/utilizatori.ts
import { Release } from "./release";

export class Utilizatori {
    idutilizator: number = 0;
    nume: string = '';
    email: string = '';
    esteadmin: number = 0;
    datacreare: Date = new Date();
    datamodificare: Date = new Date();
    releases: Release[] = [];
}
